finder_test.go

```diff
@@ -54,8 +54,8 @@ func TestNonExistentDirectory(t *testing.T) {
 
 func TestParser(t *testing.T) {
 	entities, errs, err := FindEntities([]string{"."}, []string{})
-	assert.Equal(t, 13, len(entities), fmt.Sprintf("%s", entities))
-	assert.Equal(t, 14, len(errs), fmt.Sprintf("%v", errs))
+	assert.Equal(t, 15, len(entities), fmt.Sprintf("%s", entities))
+	assert.Equal(t, 17, len(errs), fmt.Sprintf("%v", errs))
```

Comment:
I think you added 4 negative tests and one positive, so these do not add up correctly.